seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/sink/JdbcSink.java

```diff
@@ -194,7 +194,10 @@ public Optional<SaveModeHandler> getSaveModeHandler() {
                 return Optional.empty();
             }
             Optional<Catalog> catalogOptional =
-                    JdbcCatalogUtils.findCatalog(jdbcSinkConfig.getJdbcConnectionConfig(), dialect);
+                    JdbcCatalogUtils.findCatalog(
+                            jdbcSinkConfig.getJdbcConnectionConfig(),
+                            dialect,
+                            jdbcSinkConfig.isCreateIndex());
```

Comment:
Why catalog should care aboult `createIndex` value at now?

Reply:
> createIndex

Because the createTable method comes from catalog.